packages/twenty-front/src/modules/settings/data-model/object-details/components/SettingsObjectFieldItemTableRow.tsx

```diff
@@ -60,6 +60,18 @@ export const SettingsObjectFieldItemTableRow = ({
 
   const fieldType = fieldMetadataItem.type;
   const isFieldTypeSupported = isFieldTypeSupportedInSettings(fieldType);
+  useEffect(() => {
```

Comment:
This logic is not necessary because we already have a fieldMetadataItem array higher up in the tree, the row shouldn't be related to this feature.

Reply:
Removed the unwanted logic and used the actual value in higher level component instead